sprockets/mixins/sentry/__init__.py

```diff
@@ -24,7 +24,15 @@
 
 LOGGER = logging.getLogger(__name__)
 SENTRY_CLIENT = 'sentry_client'
-URI_RE = re.compile(r'^[\w\+\-]+://.*:(\w+)@.*')
+
+# This matches the userinfo production from RFC3986 with some extra
+# leniancy to account for poorly formed URLs.  For example, it lets
+# you include braces and other things in the password section.
+URI_RE = re.compile(r"^[\w\+\-]+://"
+                    r"[-a-z0-9!$&'()*+,;=%.]+:"
```

Comment:
Just a FYI, underscore is not included in the `username` part...probably want in many cases?